packages/components/src/DataList/DataList.types.ts

```diff
@@ -72,4 +73,13 @@ export interface DataListProps<T extends DataListObject> {
    * in the list
    */
   readonly totalCount?: number | null;
+
+  /**
+   * Determine if the header is visible at a given breakpoint. If one isn't provided,
+   * it will use the value from the next smallest breakpoint that has a value.
+   * @default { xs: true, sm: true, md: true, lg: true, xl: true }
+   */
+  readonly headerVisibility?: { [Breakpoint in Breakpoints]?: boolean };
 }
+
+export { Breakpoints } from "./DataList.const";
```

Comment:
gotta admit, did not expect this to be down here. Consistency-wise, should it be down here?